diff -u -r -N a/meson.build b/meson.build
--- a/meson.build	2017-06-30 17:20:30.000000000 -0400
+++ b/meson.build	2017-07-27 08:14:15.767156051 -0400
@@ -71,8 +71,10 @@
 if not get_option('disable-help')
     subdir('help')
 endif
-subdir('tests')
-subdir('tests/validate-tests')
+if get_option('enable-tests')
+    subdir('tests')
+    subdir('tests/validate-tests')
+endif
 
 archiver = find_program('git-archive-all', required : false)
 if archiver.found()
