--- a/xenserver/etc_xapi.d_plugins_openvswitch-cfg-update	2015-06-18 19:32:47.000000000 +0000
+++ b/xenserver/etc_xapi.d_plugins_openvswitch-cfg-update.new	2015-07-08 13:21:29.227231479 +0000
@@ -44,6 +44,10 @@
 def update(session, args):
     # Refresh bridge network UUIDs in case this host joined or left a pool.
     script = '/opt/xensource/libexec/interface-reconfigure'
+    # NOTE(jamespage): Override with distro xcp xapi locations
+    # if present
+    if os.path.exists('/usr/lib/xcp/lib/interface-reconfigure'):
+        script = '/usr/lib/xcp/lib/interface-reconfigure'
     try:
         retval = subprocess.call([script, 'rewrite'])
         if retval != 0:
