--- a/libcontainerd/remote_linux.go	2016-04-27 12:01:07.486000000 +0000
+++ b/libcontainerd/remote_linux.go	2016-04-27 12:01:41.100000000 +0000
@@ -30,7 +30,7 @@
 	maxConnectionRetryCount   = 3
 	connectionRetryDelay      = 3 * time.Second
 	containerdShutdownTimeout = 15 * time.Second
-	containerdBinary          = "docker-containerd"
+	containerdBinary          = "containerd"
 	containerdPidFilename     = "docker-containerd.pid"
 	containerdSockFilename    = "docker-containerd.sock"
 	eventTimestampFilename    = "event.ts"
@@ -349,7 +349,7 @@
 	}
 
 	// Start a new instance
-	args := []string{"-l", r.rpcAddr, "--runtime", "docker-runc", "--metrics-interval=0"}
+	args := []string{"-l", r.rpcAddr, "--runtime", "runc", "--metrics-interval=0"}
 	if r.debugLog {
 		args = append(args, "--debug")
 	}
