--- a/libcontainerd/remote_linux.go	2016-05-05 05:27:26.458000000 +0000
+++ b/libcontainerd/remote_linux.go	2016-05-05 05:28:21.825000000 +0000
@@ -30,7 +30,7 @@
 	maxConnectionRetryCount   = 3
 	connectionRetryDelay      = 3 * time.Second
 	containerdShutdownTimeout = 15 * time.Second
-	containerdBinary          = "docker-containerd"
+	containerdBinary          = "containerd"
 	containerdPidFilename     = "docker-containerd.pid"
 	containerdSockFilename    = "docker-containerd.sock"
 	eventTimestampFilename    = "event.ts"
@@ -349,7 +349,7 @@
 	}
 
 	// Start a new instance
-	args := []string{"-l", r.rpcAddr, "--runtime", "docker-runc"}
+	args := []string{"-l", r.rpcAddr, "--runtime", "runc"}
 	if r.debugLog {
 		args = append(args, "--debug", "--metrics-interval=0")
 	}
