diff -u -r -N a/examples/eqNBody/client.cpp b/examples/eqNBody/client.cpp
--- a/examples/eqNBody/client.cpp	2015-11-06 07:13:36.000000000 -0500
+++ b/examples/eqNBody/client.cpp	2016-03-16 12:15:48.813570538 -0400
@@ -73,9 +73,6 @@
         disconnectServer( _server );
         return EXIT_FAILURE;
     }
-    else if( _config->getError( ))
-        LBWARN << "Error during initialization: " << _config->getError()
-               << std::endl;
 
     return EXIT_SUCCESS;
 }
diff -u -r -N a/examples/eqNBody/config.cpp b/examples/eqNBody/config.cpp
--- a/examples/eqNBody/config.cpp	2015-11-06 07:13:36.000000000 -0500
+++ b/examples/eqNBody/config.cpp	2016-03-16 12:05:36.882214074 -0400
@@ -71,7 +71,7 @@
     deregisterObject( &_initData );
     deregisterObject( &_frameData );
 
-    _initData.setFrameDataID( 0 );
+    _initData.setFrameDataID( eq::uint128_t( ));
 }
 
 bool Config::loadInitData( const eq::uint128_t& id )
diff -u -r -N a/examples/eqNBody/frameData.cpp b/examples/eqNBody/frameData.cpp
--- a/examples/eqNBody/frameData.cpp	2015-11-06 07:13:36.000000000 -0500
+++ b/examples/eqNBody/frameData.cpp	2016-03-16 12:05:45.370260633 -0400
@@ -153,7 +153,7 @@
             return getProxyVersion( i );
     }
 
-    return 0;
+    return eq::uint128_t( );
 }
 
 void FrameData::toggleStatistics()
diff -u -r -N a/examples/eqNBody/initData.cpp b/examples/eqNBody/initData.cpp
--- a/examples/eqNBody/initData.cpp	2015-11-06 07:13:36.000000000 -0500
+++ b/examples/eqNBody/initData.cpp	2016-03-16 12:05:53.930307588 -0400
@@ -47,7 +47,7 @@
     
     InitData::~InitData()
     {
-           setFrameDataID( 0 );
+           setFrameDataID( eq::uint128_t( ));
     }
     
     void InitData::getInstanceData( co::DataOStream& os )
