diff --git a/libcfs/autoconf/lustre-libcfs.m4 b/libcfs/autoconf/lustre-libcfs.m4
index b2ba484dc0..7ece383cf5 100644
--- a/libcfs/autoconf/lustre-libcfs.m4
+++ b/libcfs/autoconf/lustre-libcfs.m4
@@ -680,6 +680,24 @@ get_user_pages_gup_flags, [
 ])
 ]) # LIBCFS_GET_USER_PAGES_GUP_FLAGS
 
+#
+# Kernel version 4.4.168 commit 8e50b8b07f462ab4b91bc1491b1c91bd75e4ad40
+# mm: replace get_user_pages() write/force parameters with gup_flags (backport)
+#
+AC_DEFUN([LIBCFS_GET_USER_PAGES_GUP_FLAGS_BACKPORT], [
+LB_CHECK_COMPILE([if 'get_user_pages()' takes gup_flags in arguments - backport],
+get_user_pages_gup_flags_backport, [
+	#include <linux/mm.h>
+],[
+	int rc;
+	rc = get_user_pages(0, 0, 0, 0, FOLL_WRITE, NULL, NULL);
+],[
+	AC_DEFINE(HAVE_GET_USER_PAGES_GUP_FLAGS_BACKPORT, 1,
+		[get_user_pages takes gup_flags in arguments - backport])
+])
+]) # LIBCFS_GET_USER_PAGES_GUP_FLAGS_BACKPORT
+
+
 #
 # Kernel version 4.10 commit 7b737965b33188bd3dbb44e938535c4006d97fbb
 # libcfs: Convert to hotplug state machine
@@ -764,6 +782,8 @@ LIBCFS_HAVE_CPUMASK_PRINT_TO_PAGEBUF
 # 4.2
 LIBCFS_HAVE_TOPOLOGY_SIBLING_CPUMASK
 LIBCFS_FPU_API
+# 4.4
+LIBCFS_GET_USER_PAGES_GUP_FLAGS_BACKPORT
 # 4.5
 LIBCFS_CRYPTO_HASH_HELPERS
 # 4.6
diff --git a/libcfs/libcfs/linux/linux-curproc.c b/libcfs/libcfs/linux/linux-curproc.c
index 7b2e46e61b..c3d30975a0 100644
--- a/libcfs/libcfs/linux/linux-curproc.c
+++ b/libcfs/libcfs/linux/linux-curproc.c
@@ -151,6 +151,8 @@ static int cfs_access_process_vm(struct task_struct *tsk,
 
 #if defined(HAVE_GET_USER_PAGES_GUP_FLAGS)
 		rc = get_user_pages(addr, 1, write ? FOLL_WRITE : 0, &page, &vma);
+#elif defined(HAVE_GET_USER_PAGES_GUP_FLAGS_BACKPORT)
+		rc = get_user_pages(tsk, mm, addr, 1, write ? FOLL_WRITE : 0, &page, &vma);
 #elif defined(HAVE_GET_USER_PAGES_6ARG)
 		rc = get_user_pages(addr, 1, write, 1, &page, &vma);
 #else
